project/src/main/java/com/google/sps/ensemble/data/README

```diff
@@ -0,0 +1 @@
+This directory is a placeholder for backend Java classes and other data.
```

Comment:
You probably don't need a README in every package. Outside the one at `java/com/google/sps/ensemble/README`, I think you can remove the one here and in under `servlets/`.